src/queue/dune

```diff
@@ -4,26 +4,31 @@
 (alias
  (name default)
  (package multicoretests)
- (deps lin_tests.exe))
+ (deps lin_tests_dsl.exe))
 
 (env
  (_
   (binaries
    (../check_error_count.exe as check_error_count))))
 
 
+(executable
+ (name lin_tests_dsl)
+ (modules lin_tests_dsl)
+ (flags (:standard -w -27))
+ (libraries qcheck lin))
+
 (executable
  (name lin_tests)
  (modules lin_tests)
  (flags (:standard -w -27))
- (libraries qcheck lin)
- (preprocess (pps ppx_deriving_qcheck ppx_deriving.show ppx_deriving.eq)))
+ (libraries qcheck lin))
```

Comment:
I think the preprocessing has been mistakingly removed from the old non-DSL rule.